apps/tailwind-components/assets/css/plugins/datepicker.css

```diff
@@ -0,0 +1,383 @@
+/* override header buttons */
+.dp__main .dp--header-wrap .dp__month_year_wrap .dp__btn.dp__month_year_select {
+  @apply text-input font-display tracking-widest uppercase;
+}
+
+.dp__main
+  .dp--header-wrap
+  .dp__month_year_wrap
+  .dp__btn.dp__month_year_select:hover {
+  @apply text-input bg-transparent;
+}
```

Comment:
I didn't apply the button background color here. There isn't a good style for this button that works in all themes. I did not want to create an additional tailwind class for this either.